.github/workflows/_linux-test.yml

```diff
@@ -253,6 +253,12 @@ jobs:
             TEST_COMMAND=.ci/pytorch/test.sh
           fi
 
+          # Leaving 1GB for the runner and other things
```

Comment:
Wondering if 1 GB might be an excessive amount to reserve for the runner...

We can stick with this one for now but should monitor the usage later and see if this can be scaled back